src/scheduler/MpiWorld.cpp

```diff
@@ -261,16 +262,51 @@ std::string MpiWorld::getHostForRank(int rank)
     return host;
 }
 
+// Returns a pair (sendPort, recvPort)
```

Comment:
Also worth checking this which has changed since last review.

Reply:
(basically [this commit](https://github.com/faasm/faabric/pull/111/commits/88bcb7d6a6c88f8118301d8948fd9b39e40f3acc))